### PR TITLE
Use `layout opam` in `.envrc` in opam path doc

### PR DESCRIPTION
**Pull request by smorimoto**
- https://github.com/direnv/direnv/pull/1298

### CODE REVIEW

File: data/tutorials/platform/0_09_opam_path.md
Comment (smorimoto) [inline data/tutorials/platform/0_09_opam_path.md:17]: https://www.shellcheck.net/wiki/SC2046

File: data/tutorials/platform/0_09_opam_path.md
Comment (smorimoto) [inline data/tutorials/platform/0_09_opam_path.md:50]: https://www.shellcheck.net/wiki/SC2046

**Review by cuihtlauac — CHANGES_REQUESTED** [review]
Hi @smorimoto, thanks for adding support for opam in direnv; this is great. I love that tool. Could you add a note on direnv version and use case? It will take time until your direnv patch lands in distros. In the interim, we need to support both syntaxes.

Also, I've merged the changes unrelated to direnv in a separate PR: #2602. You'll have to rebase

**Review by christinerose — APPROVED** [review]

**Comment by sabine** [comment]
Is this ready to merge?

**Comment by smorimoto** [comment]
Not yet! We have to wait for the next direnv release 🥲

**Comment by tmattio** [comment]
As I understand, this will only be compatible with the last version of direnv. Even when it is released, we can't assume the user will have this version, so I think it's a bit too early to recommend users to use this.

**Comment by smorimoto** [comment]
Released: https://github.com/direnv/direnv/releases/tag/v2.35.0

**Comment by smorimoto** [comment]
https://repology.org/project/direnv/versions

**Pull request closed without merging**
